app/views/recipes/show.html.haml

```diff
@@ -0,0 +1,24 @@
+.page-header
+  %h1.text-uppercase.text-center Recipes Details
+
+.row
+  .col-md-4.col-md-offset-4
+    %p
+      %strong Name: 
+      #{@recipe.name}  
+    %p
+      %strong Preparation:
+      %br
+      #{@recipe.preparation}
+    %p
+      %strong Created On: 
+      #{@recipe.created_at.strftime('%d %B %Y')}
```

Comment:
Was looking into the above link, should i have to install the gem delocalize. I try to add the format in the en.yml but no sure how to make it work :(